python/prophet/diagnostics.py

```diff
@@ -630,6 +630,7 @@ def smape(df, w):
     Dataframe with columns horizon and smape.
     """
     sape = np.abs(df['y'] - df['yhat']) / ((np.abs(df['y']) + np.abs(df['yhat'])) / 2)
+    sape = sape.fillna(0)
```

Comment:
are there tests covering these methods?

Reply:
There was a test looking at the `mape` metric underneath the wider `test_performance_metrics()` method, so I have extended that to test for zero values across all other metrics.

Reply:
Great nice one